lib/hammer_cli_foreman.rb

```diff
@@ -107,6 +107,10 @@ def self.exception_handler_class
       'HammerCLIForeman::SmartClassParameter', 'hammer_cli_foreman/smart_class_parameter'
     )
 
+    HammerCLI::MainCommand.lazy_subcommand('ssh-keys', _("Manipulate smart class parameters."),
```

Comment:
I hope it's not manipulating smart class parameters ;-)

Reply:
It's "Managing User SSH Keys.". corrected. :)